libraries/anaconda3-solaris-0.1.2/Dockerfile

```diff
@@ -1,3 +1,4 @@
-COPY --chown=0:0 anaconda3-solaris-0.1.2/context/solaris-build /usr/local/bin/solaris-build
-RUN git clone --single-branch --branch v0.1.2 https://github.com/CosmiQ/solaris.git /tmp/solaris && \
-    /usr/local/bin/solaris-build /tmp/solaris
+ENV REQFILE=/home/algo/dependency-solaris-0.1.2.yml
+COPY --chown=0:0 anaconda3-solaris-0.1.2/context/build /usr/local/bin/dependency-build
```

Comment:
can you name this something more specific to `solaris`? it feels weird to install a solaris-specific script to `/usr/local/bin` with a gneric name like this

Reply:
oh for sure, I didn't think the name would matter as it's just getting used once and thrown away, we could throw these in /tmp

Reply:
i don't see where the file would be getting thrown away, i think it would still be available in the output image (and since the file was `COPY`'d even deleting the file would only result in a tombstone marker)

Reply:
I moved to /tmp, it wouldn't be visible at the output image as these ops are done in the buildtime and aren't migrated. Let me know if that works

Reply:
i'm not sure i follow - ` it wouldn't be visible at the output image as these ops are done in the buildtime` the file would definitely be visible in the complete `langpack-builder` or `langpack-runner` image.  if the package is only installed in the buildtime image then it wouldn't be in the final algorithm image, but it's still a weird file to have sitting around that could be overwritten by any other package that writes data to `/tmp`.  i would just want the file named like `anaconda-solaris-build` or something, not `dependency-build`, regardless of the directory it's in, and putting in `/tmp` doesn't really make a difference since the file is left around anyway i think it's fine to be in `/usr/local/bin`, just need a unique filename